HotorNot/app/src/main/java/com/example/hotornot/fragments/MainScreenFragment.kt

```diff
@@ -30,6 +31,16 @@ class MainScreenFragment : Fragment() {
         binding.btnLeft.setOnClickListener {
             binding.imageView.setImageResource(R.drawable.georgi)
             binding.txtName.text = R.string.georgi.toString()
+
+        }
+    }
+
+    private fun makeBtnDisappear(){
+        if(binding.txtName.text == "Georgi"){
```

Comment:
When you have only one statement you can remove braces from all if statement 
 if(binding.txtName.text == "Georgi") binding.btnLeft.visibility
  else if (binding.txtName.text == "Stan") " binding.btnRight.visibility